tests/src/Interop/PInvoke/Array/MarshalArrayAsParam/AsDefault/AsDefaultTest.cs

```diff
@@ -244,12 +244,19 @@ public struct TestStruct
     [DllImport("MarshalArrayLPArrayNative")]
     private static extern bool CStyle_Array_Bool_Out(
         [Out] bool[] actual, int cActual);
+
+    [DllImport("MarshalArrayLPArrayNative")]
+    private static extern int Get_Multidimensional_Array_Sum(int[,] array, int rows, int columns);
```

Comment:
Are there any relevant `MarshalAs` options that we could validate here?

Reply:
If we wanted to we could add additional tests for SafeArray tests, but the marshaler classification doesn't actually distinguish between a multidimensional array and an SZArray when looking at the `MarshalAs` information.